python/cudf/cudf/tests/test_stats.py

```diff
@@ -201,13 +202,25 @@ def test_approx_quantiles_int():
 
 
 @pytest.mark.parametrize("data", [[], [1, 2, 3, 10, 326497]])
-@pytest.mark.parametrize("q", [[], 0.5, 1, 0.234, [0.345], [0.243, 0.5, 1]])
+@pytest.mark.parametrize(
+    "q",
+    [
+        [],
+        0.5,
+        1,
+        0.234,
+        [0.345],
+        [0.243, 0.5, 1],
+        np.array([0.5, 1]),
+        cp.array([0.5, 1]),
+    ],
+)
 def test_misc_quantiles(data, q):
 
     pdf_series = cudf.utils.utils._create_pandas_series(data=data)
     gdf_series = cudf.Series(data)
 
-    expected = pdf_series.quantile(q)
+    expected = pdf_series.quantile(q.get() if isinstance(q, cp.ndarray) else q)
```

Comment:
Isn't the whole point of this PR to support cupy arrays? Why are we converting it to a numpy array in the test?

Reply:
This was because `pandas` doesn't support it :( 
```python
>>> s.quantile(cp.arrany([0,1]))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/nvme/0/pgali/envs/cudfdev/lib/python3.8/site-packages/cupy/__init__.py", line 901, in __getattr__
    raise AttributeError(f"module 'cupy' has no attribute {name!r}")
AttributeError: module 'cupy' has no attribute 'arrany'
>>> s.quantile(cp.array([0,1]))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/nvme/0/pgali/envs/cudfdev/lib/python3.8/site-packages/pandas/core/series.py", line 2442, in quantile
    validate_percentile(q)
  File "/nvme/0/pgali/envs/cudfdev/lib/python3.8/site-packages/pandas/util/_validators.py", line 408, in validate_percentile
    q_arr = np.asarray(q)
  File "cupy/_core/core.pyx", line 1381, in cupy._core.core.ndarray.__array__
TypeError: Implicit conversion to a NumPy array is not allowed. Please use `.get()` to construct a NumPy array explicitly.
```

Reply:
Oh I didn't see that you're doing this only for the `pdf`, not the `gdf`. :+1: